libkbfs/test_stallers.go

```diff
@@ -157,6 +157,7 @@ func (s *NaïveStaller) StallMDOp(stalledOp StallableMDOp, maxStalls int,
 			},
```

Comment:
Looks like the comment above `// Stall the delegate server as well` should drop the `as well`, right?